models/place-terms.json

```diff
@@ -31,23 +40,14 @@
         "example": "placetype",
         "description": "The domain of the term. Can be one of `placetype`, `theme`, or `facility`.",
         "enum": [
-          "placetype",
           "theme",
-          "facility"
+          "facility",
+          "eventtype"
```

Comment:
Places can also not have a `theme` I recently learned. https://github.com/cultuurnet/udb3-silex/pull/779